core/src/main/java/com/rudderstack/android/sdk/core/RudderDeviceModeManager.java

```diff
@@ -100,10 +104,16 @@ private List<RudderServerDestination> getConsentedDestinations(RudderServerConfi
 
     }
 
-    private void setDestinationsWithTransformationsEnabled(List<RudderServerDestination> destinations) {
+    // The sourceConfig will be used to generate two lists: one containing destinations with enabled transformations,
+    // and another containing destinations to be excluded in case of a transformation error.
+    private void setDestinations(List<RudderServerDestination> destinations) {
```

Comment:
I think this should be renamed to `segregateDestinations`

Reply:
Done.